lib/libraries/index.js

```diff
@@ -307,6 +326,18 @@ function defineC3(){
                 if (this.config.tooltip && this.config.tooltip.format && this.config.tooltip.format.value)
                 {
                   valueFormatted = this.config.tooltip.format.value.call(this, valueFormatted, ratio, id, index);
+                  // Restore value from percents calculation for stacking 100% charts
+                  if (options.stacking === 'percent' && (type === 'bar' || type === 'horizontal-bar' || type === 'area' 
+                  || type === 'area-step' || type === 'area-spline')) {
+                    valueFormatted = parseFloat(((valueFormatted / 100) * sumArray[index]).toFixed(2));
+                  }
+                  return valueFormatted;
+                }
+                // Restore value from percents calculation for stacking 100% charts
+                if (options.stacking === 'percent' && (type === 'bar' || type === 'horizontal-bar' || type === 'area' 
+                || type === 'area-step' || type === 'area-spline')) {
+                  valueFormatted = ((valueFormatted / 100) * sumArray[index]).toFixed(2);
+                  return parseFloat(valueFormatted);
```

Comment:
let's remove :329 - :333 , and `return` from :334, so we don't have to duplicate this code